### DISCUSSION TITLE
AzureAD Authentication

### DISCUSSION THREAD

**Discussion by RyanJ-MP** [requarks/wiki#5300] [Error / Bug Report]
Hi,

I followed the instructions here (https://docs.requarks.io/auth/azure) for AzureAD auth. When we click the Login with AzureAD button we get an Oops, something went wrong... Invalid authentication provider. error.

Any ideas what I can try to get this working?

Operating System
Docker Container (Linux)
Current Version
2.5.282

**Comment by ghost** [comment]
It is not quite evident from the documentation, but you need to go to your AAD Authentication Strategy in your Wiki.js and scroll down. There you will see a Callback URL / Redirect URI. Use this instead of https://YOUR-WIKI.DOMAIN.COM/login/azure/callback . Here "azure" will be replaced with a high amount of characters. Then it should work.

**Reply by RyanJ-MP** [reply]
hi @Aemilius11,
This was one of the marked solutions that I had found when googling, unfortunately, this did not work for me.
